app/src/providers/web3/contract/airdrop-contract/airdrop-contract.ts

```diff
@@ -24,10 +24,9 @@ export class AirdropContract {
     }
   }
 
-  async release(payee: string) {
-    const receipt = await this.contract.methods.release(payee).send({ from: payee });
-
-    return receipt;
+  release(payee: string) {
+    const eventEmitter = this.contract.methods.release(payee).send({ from: payee });
+    return eventEmitter;
```

Comment:
```suggestion
    const eventEmitter = this.contract.methods.release(payee).send({ from: payee });
    
    return eventEmitter;
```

I think you haven't installed `eslint` and `prettier` VSCode extensions.